ui/libproject/devstart.rb

```diff
@@ -56,23 +56,21 @@ def dev_up(*args)
 
   ensure_git_hooks
 
-  unless Dir.exist?("node_modules")
-    install_dependencies
-  end
+  install_dependencies
```

Comment:
Will this start a Docker container, do the install, and then kill that container before continuing on to the main startup? How is it for speed? I'm wondering if doing 'npm install' inside the main UI Docker container would be faster, if this requires more/extra setup.

Reply:
I believe this actually does this on the host machine, but I'm not certain. I can do some more investigation there and get back to this.

Reply:
The answer is that it actually does it inside the docker image, and that translates back down to the host machine. It seems pretty fast when I run it on my machine. (I didn't super notice the difference in speed.)